resources/views/page-builder/faq.blade.php

```diff
@@ -1,20 +1,20 @@
-<div class="relative z-10 bg-gray-100 pt-16 sm:pt-32"{{ ($component_id ?? false) ? " id=" . $component_id : "" }}>
-    <div class="max-w-7xl mx-auto px-6 sm:px-6 w-full z-10 relative">
-        @if(($title_heading ?? false) || ($title_subheading ?? false))
-            <h2 class="text-3xl font-extrabold tracking-tight sm:text-4xl text-center">
-                @if($title_heading ?? false)
-                    <span class="bg-clip-text text-transparent bg-gradient-to-b from-primary-100 to-primary-200 pb-2">{{ $title_heading }}</span>
+<div class="faq relative z-10 bg-gray-100 component pt-16 sm:pt-32"{{ $component_id ?? false ? ' id=' . $component_id : '' }}>
```

Comment:
I see the custom classes scattered in the `class` attribute in some components, don't we want that grouped in either the front or end of the attribute? Happens in a few components

Reply:
Yeah i left the same comment, but it has been marked as resolved, but i don't see anything being resolved. [link](https://github.com/rapidez/website/pull/34#discussion_r1803099102)